IKZ_plugin/src/ikz_plugin/movpe/movpe2/utils.py

```diff
@@ -122,6 +120,9 @@ def dict_nan_equal(dict1, dict2):
 def create_archive(
     entry_dict, context, filename, file_type, logger, *, overwrite: bool = False
 ):
+    from nomad.datamodel.context import ClientContext, ServerContext
```

Comment:
Again, is there a reason for moving the imports here? You have the same imports again in the next function.